python_docs_theme/static/pydoctheme.css

```
@@ -29,6 +29,11 @@ pre {
     color: inherit;
 }
 
+a {
```

```suggestion
a:any-link {
```

OR

```suggestion
a[href] {
```

Let's only underline actual links rather than [placeholder links](https://www.enovate.co.uk/blog/2013/07/16/html5-placeholder-links).

https://developer.mozilla.org/en-US/docs/Web/CSS/:any-link